lib/cosmos/tools/cmd_sequence/cmd_sequence.rb

```diff
@@ -267,7 +267,11 @@ def initialize_central_widget
     # Export the sequence list into a custom binary format
     def file_export
       return if @sequence_list.empty? || @exporter.nil?
-      @exporter.export(@filename, @sequence_dir, @sequence_list)
+      begin
+        @exporter.export(@filename, @sequence_dir, @sequence_list)
+      rescue => error
+        Qt::MessageBox.critical(self, 'Export Error', "The #{@exporter.class} raised the folllowing error:\n#{error.message}")
```

Comment:
You should either use ExceptionDialog or include the backtrace.  I hate trying to debug things and not knowing what line to look at.